react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import './HotButton.css';
+
+export default function HotButton({ temperature }) {
+  const [timesClicked, setTimesClicked] = useState(0);
+  function handleClick() {
+    setTimesClicked(timesClicked + 1);
+  }
+
+  if (timesClicked < 3) {
+    temperature = 'freezing';
+  } else if (timesClicked < 6) {
+    temperature = 'cold';
+  } else if (timesClicked < 9) {
+    temperature = 'luke-warm';
+  } else if (timesClicked < 12) {
+    temperature = 'warmer';
+  } else if (timesClicked < 15) {
+    temperature = 'hot';
+  } else if (timesClicked < 18) {
+    temperature = 'on-fire';
+  }
```

Comment:
You never initialize your `temperature` variable

Reply:
fixed!